vendor/bat-native-ads/src/bat/ads/internal/search_providers.h

```diff
@@ -41,6 +41,7 @@ const std::vector<SearchProviderInfo> _search_providers = {
       false),
   SearchProviderInfo(
       "Google",
+      // TODO(https://github.com/brave/brave-browser/issues/8487)
```

Comment:
nit: synopsis of what 8484 is, i.e. `TODO(https://github.com/brave/brave-browser/issues/8487): foobar`